components/kueue/production/stone-prd-rh01/queue-config/cluster-queue.yaml

```diff
@@ -30,6 +31,8 @@ spec:
         nominalQuota: 500Ti
       - name: aws-ip
         nominalQuota: '250'
+      - name: mintmaker
+        nominalQuota: '200'
```

Comment:
We can probably set it to `150` for rh01 cluster. Setting it to `100` might also work, but to be safe, let's start with `150`. It usually takes about 1h30m to complete all the pipelineruns. With the limit changed to `150`, I expect all pipelineruns should be able to finish within 3 hours.